T7411: Uncaught exception ghc-internal:GHC.Internal.Exception.ErrorCall:

Prelude.undefined
CallStack (from HasCallStack):
  undefined, called at T7411.hs:3:25 in main:Main

HasCallStack backtrace:
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:92:13 in ghc-internal:GHC.Internal.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:127:5 in ghc-internal:GHC.Internal.Exception
  undefined, called at T7411.hs:3:25 in main:Main

